inst/shiny/dashboard/global.R

```diff
@@ -39,12 +44,12 @@ connection_string <- paste0("Driver={", odbc_conf$driver,
 # or by reading in SQL queries, and using your `connection_string` above to connect
 # to a database.
 
-# CSV:
-# my_csv_data <- read_csv("CSV/filename.csv")
+# Data:
+# my_csv_data <- read_csv("Data/filename.csv")
 
-# SQL:
+# Queries:
 # my_connection <- odbcDriverConnect(connection_string)
-# my_query <- paste(readLines("SQL/filename.sql"), sep = "\n")
+# my_query <- read_sql_script("SQL/filename.sql")
```

Comment:
I'd do `dfeR::read_sql_script()` to make it more obvious that this is something else in the package. Possibly add a short comment here to explain `::` notation?

Reply:
Was thinking about this - would you do library and this? I find if people aren’t using packrat that scanning library is the easiest way to get the packages you need? Again this might be one for training